x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/add_domain/validation_step_panel.test.tsx

```diff
@@ -45,7 +49,22 @@ describe('ValidationStepPanel', () => {
       );
       expect(wrapper.find('[data-test-subj="action"]')).toHaveLength(1);
     });
-    it('does not display the passed error message or action when state is not invalid', () => {
+
+    it('displays the passed error message and action is warning', () => {
```

Comment:
Should this have been

```suggestion
    it('displays the passed error message and action when state is warning', () => {
```

?